pyblazing/pyblazing/apiv2/context.py

```diff
@@ -151,19 +165,21 @@ def initializeBlazing(ralId=0, networkInterface='lo', singleNode=False,
         workerIp.encode(),
         ralCommunicationPort,
         singleNode,
-        config_options)
-    
-    if (os.path.isabs(logging_dir_path)):
+        config_options,
+    )
+
+    if os.path.isabs(logging_dir_path):
         log_path = logging_dir_path
     else:
         log_path = os.path.join(os.getcwd(), logging_dir_path)
 
     return ralCommunicationPort, workerIp, log_path
 
+
 def getNodePartitionKeys(df, client):
-    from dask import delayed
+    # from dask import delayed
```

Comment:
if we are not using this we should just delete it

Reply:
removed